modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -1,4 +1,9 @@
 import Base from '../../../../../../assets/dev/js/frontend/handlers/base';
```

Comment:
This is existing code, but please try to change this to

```suggestion
import Base from 'elementor/assets/dev/js/frontend/handlers/base';
```

We should be using an alias if possible.